README.md

```diff
@@ -16,43 +16,50 @@ Installation requires the following steps:
 1. Set up the required Active Directory security permissions for the application to authorize it to read threat intelligence data and activity reports for your organization.
 1. Create an Access Key that will allow the application to connect to the Alert Logic Cloud Defender and Cloud Insight backend.
 1. Download and deploy a custom ARM template to Microsoft Azure to create functions for collecting and managing O365 log data
-1. Verify that installation was successful using Alertlogic CloudDefender UI.
+1. Verify that installation was successful using the Alertlogic UI under Configuration -> Deployments -> All Deployments -> Log Sources -> Filter by "Office 365" collection method.
 
 ## Register a New O365 Web Application in O365
 
 In order to install O365 Log collector:
 
 1. Log into [O365 portal](https://portal.office.com) as AD tenant administrator.
 1. Navigate to `Admin Centers` and `Azure AD`.
-1. On the left side panel click `Azure Active Directory` and `App Registrations`.
-1. Click `+New application registration`, fill in configuration parameters and click `Create`:
+1. On the left side panel click `Azure Active Directory` then select `App Registrations`.
+1. Click `+ New application registration`, fill in configuration parameters and click `Create`:
     * `Name` - for example `alo365collector`.
     * Select `Web app/ API` as `Application type`.
-    * In `Sign-on URL` enter some URL, for example `http://alo365collector.com`. **Note**, it is not used anywhere.
+    * In `Sign-on URL` enter some URL, for example `http://alo365collector.com`. **Note**, it is not used anywhere within your subscription.
 
```

Comment:
IIRC, this value is required by Azure, but not used anywhere / just ignored. I'm not sure if it's validated here, but maybe this should say something like "this value will be ignored, but a valid URL is required by the Azure web app registration form"?